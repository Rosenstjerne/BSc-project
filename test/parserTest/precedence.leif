var int a;

a = 2 + 5 * 3 + 2; # 17