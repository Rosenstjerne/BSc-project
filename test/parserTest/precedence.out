function null main() {
	var int a;
	
	a = ((2 + (5 * 3)) + 2);
	
}
